app/components-react/editor/elements/mixer/GLVolmeters.tsx

```diff
@@ -133,12 +138,6 @@ class GLVolmetersModule {
     });
   }
 
-  // update volmeters subscriptions when audio sources change
-  watchAudioSources = injectWatch(
-    () => this.audioSources,
-    () => this.subscribeVolmeters(),
-  );
-
```

Comment:
I don't think we can simply remove this watcher? We still need the volmeters to update when the audio sources change, unless I'm missing something here.